src/aura/STG_CMP_Addr/STG_CMP_AddrHelper.js

```diff
@@ -0,0 +1,17 @@
+({
+	runBackfill : function (component, event, helper) {
+		// console.log();
+		var runBatchAction = component.get("c.executeEthnicityRaceBatch");
+		runBatchAction.setCallback(this, function(response) {
+	        if(response.getState() === "SUCCESS") {
+
+			    var backfillBtn = component.find("ethnicRaceMsg");
```

Comment:
Is this hiding the message or the button? The variable seems to refer to a button, but the code is searching for a message.